### ISSUE TITLE
Custom SVG Icon selection modal hangs (500 error)

### ISSUE THREAD

**Issue by phillips-jon** [engram-design/CPNav#114]
### Describe the bug

When clicking the "+ Choose" button under the "Custom Icon" section in the modal, the modal hangs and the console throws a 500 error.

### Steps to reproduce

1. Click the "+ Choose" button in the "Custom Icon" section of the modal.
2. Check the console for the error.
<img width="1440" alt="Screen Shot 2022-09-28 at 08 41 40" src="https://user-images.githubusercontent.com/2441967/192809862-8b78a638-5948-4ee5-acc2-2e934faed33c.png">
<img width="365" alt="Screen Shot 2022-09-28 at 08 44 16" src="https://user-images.githubusercontent.com/2441967/192809875-4d0fbdd6-00e0-416b-95f0-20967af1413c.png">

### Craft CMS version

4.2.5.1

### Plugin version

4.0.6

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/cp-nav` requirement in `composer.json` to:

```
"require": {
  "verbb/cp-nav": "dev-craft-4 as 4.0.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by phillips-jon** [comment]
Thank you!

**Comment by engram-design** [comment]
Fixed in [4.0.7](https://github.com/verbb/cp-nav/releases/tag/4.0.7)
